packages/manager/modules/hub/src/components/support/controller.js

```diff
@@ -0,0 +1,56 @@
+import get from 'lodash/get';
+import { MAX_TICKETS_TO_DISPLAY } from './constants';
+import SupportTicket from './ticket.class';
+
+export default class ManagerHubSupportCtrl {
+  /* @ngInject */
+  constructor($http, $q, CORE_URLS, RedirectionService, SessionService) {
+    this.$http = $http;
+    this.$q = $q;
+    this.CORE_URLS = CORE_URLS;
+    this.RedirectionService = RedirectionService;
+    this.SessionService = SessionService;
+  }
+
+  $onInit() {
+    this.isLoading = true;
+    this.tickets = [];
+    return this.$q
+      .when(this.tickets ? this.tickets : this.fetchTickets())
```

Comment:
Hmm you're setting `this.tickets = []` above, so the test will always return true no ?